statistics/testdata/trace_suite_out.json

```diff
@@ -6,14 +6,14 @@
         "Expr": "a > 0 and a < 2",
         "Trace": [
           {
-            "TableID": 57,
+            "TableID": 59,
```

Comment:
This change is because of the introduce of the new `mysql.table_cache_meta` table, the ID increases.